Alex King/WK7HW/DAY2/ghibli-search/js/main.js

```diff
@@ -0,0 +1,153 @@
+
+console.log('Ghibli Search:')
+$(() => {
+    const url = (`https://ghibliapi.herokuapp.com/films/`)
+    let lastSpot = ''
+    function homePage(){
+        $('.output').html(`home`)
+        const top = () => {
+            $('.output').append(`<h4>Alex's top picks:</h4><div class="container"></div>`)
+        axios.get(url)
+            .then( (res) => {
+                function findFavs(title){
+                    res.data.forEach((e)=>{
+                        if (e.title.includes(title)){
+                            renderSearchResults(e)
+                        } 
+                    }) // forEach
+                }
+                favourites = ["Porco Rosso", "Howl's Moving Castle", "Ponyo", "Kiki's Delivery Service", "Spirited Away"]
```

Comment:
you don't like Princess Mononoke?!

Reply:
I love Mononoke but if I had more than 5, the page looked weird. I had to make some sacrifices! 😵